abci/types/types.proto

```diff
@@ -227,10 +228,10 @@ message BlockParams {
   int64 max_gas = 2;
 }
 
-// EvidenceParams contains limits on the evidence.
 message EvidenceParams {
   // Note: must be greater than 0
-  int64 max_age = 1;
+  int64 max_age_height = 1;
```

Comment:
this can be misinterpreted - as "stop height". maybe rename to `max_age_blocks` or `max_age_num_blocks`